### ISSUE TITLE
Backward compatibility

### ISSUE THREAD

**Issue by ifellows** [ICPI/ICPIutilities#54]
It would be nice if functions like read_msd were backward compatible. When I try to read in data that previously loaded fine, ICPIutilities now throws an error. For example on "MER_Structured_Datasets_Site_IM_FY17-19_20190920_v2_1_Nigeria.txt" I get:

```
> dat <- ICPIutilities::read_msd(mer_data_source, remove_txt = FALSE)
Error in .f(.x[[i]], ...) : object 'targets' not found  
```

**Comment by achafetz** [comment]
Thanks, Ian! Makes a ton of sense. I'll see if I can find some time later this week to address to work with the old style of MSDs

**Comment by achafetz** [comment]
resolved by #55
